velox/common/memory/MemoryPool.h

```diff
@@ -384,12 +384,13 @@ class MemoryPool : public std::enable_shared_from_this<MemoryPool> {
   /// corresponding method.
   virtual void leaveArbitration() noexcept = 0;
 
-  /// Returns how many bytes is reclaimable from this memory pool. The function
-  /// returns true if this memory pool is reclaimable, and returns the estimated
-  /// reclaimable bytes in 'reclaimableBytes'. If 'reclaimer_' is not set, the
-  /// function returns false, otherwise invoke the reclaimer's corresponding
-  /// method.
-  virtual bool reclaimableBytes(uint64_t& reclaimableBytes) const = 0;
+  /// Returns the estimated number of bytes that can be reclaimed from this
+  /// memory pool. If this memory pool is reclaimable, the function will
```

Comment:
```
function estimates ... and returns in 'reclaimableBytes'. 
If the 'reclaimer_' is not set, the function returns .... Otherwise, it invokes ...
```